openpower/package/hostboot/hostboot.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-HOSTBOOT_VERSION ?= 787d61d8918730232ac546e0b679a19723fdd70a
+HOSTBOOT_VERSION ?= 32dff4ca9db6d8b088ed6ec63f147d2aaded3e69
```

Comment:
I don't think this change was intentional.  Seems to be rolling us back to an older Hostboot.